docs/sql-data-sources-jdbc.md

```diff
@@ -286,6 +286,15 @@ logging into the data sources.
     <td>read</td>
   </tr>
 
+  <tr>
+    <td><code>maxPushDownLimit</code></td>
```

Comment:
Shall we make it a SQL config? It's not specific to any query, but more about the driver setup. A per-query option may be an overkill.

Reply:
BTW we should explain more about the targeting use case. For queries with big limit, I think Spark will OOM anyway (at the driver side).

Reply:
OK